world-server/src/world/player/Player.ts

```diff
@@ -60,10 +63,16 @@ export class Player extends Entity {
 
     respawn(){
         // TODO: respawn logic
+        this.health = this.maxHealth;
```

Comment:
Should be in a separate pull request...

Reply:
Same thing as with the initial position. It seems the guys got confused, removed.